### PR TITLE
Add 'make latexpdf' to build the PDF documentation in one step

### PR DESCRIPTION
**Pull request by asmeurer**
I have several fixes I am making to the docs build. However, some of them may more controversial or require some more review than others, so I am going to make separate PRs, as they are mostly independent fixes. 

Previously to build the pdf docs you had to run

```
make latex
export LATEXMKOPTS='-halt-on-error -xelatex'
cd _build/latex
make
```

now you can just run

```
make latexpdf
```

This was also something the output of `make latex` said you could do but didn't actually work.

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by moorepants** [comment]
LGTM

**Comment by smichr** [comment]
SPHINX is failing

File: bin/test_sphinx.sh [hunk 1]
```diff
@@ -6,12 +6,9 @@ echo "Testing SPHINX"
 cd doc
 make html
 make man
-make latex
-cd _build/latex
-export LATEXMKOPTS="-halt-on-error -xelatex -silent"
-make all || {
+make pdflatex LATEXMKOPTS="-halt-on-error -xelatex -silent" || {
```
Comment (oscargus) [inline bin/test_sphinx.sh:9]:
```suggestion
make latexpdf LATEXMKOPTS="-halt-on-error -xelatex -silent" || {
```

**Comment by oscarbenjamin** [comment]
Looks good.

There is also this:
https://github.com/sympy/sympy/blob/210636b390261ef2a13dccc541853869f3aec1e0/bin/test_travis.sh#L25-L33
and this
https://github.com/sympy/sympy/blob/210636b390261ef2a13dccc541853869f3aec1e0/release/build_docs.py#L41-L47
I guess something should also be added here:
https://github.com/sympy/sympy/blob/master/doc/src/guides/contributing/build-docs.rst

**Pull request merged**